pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -161,8 +149,18 @@ RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value \
  && curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-x86_64.sh \
  && chmod +x ~/mambaforge.sh \
  && ~/mambaforge.sh -b -p /opt/conda \
- && rm ~/mambaforge.sh \
- && /opt/conda/bin/mamba install -y -c conda-forge \
+ && rm ~/mambaforge.sh 
+
+# Install ec2 PyTorch
+RUN /opt/conda/bin/mamba install -y pytorch=${PYTORCH_VERSION} \
+  pytorch-cuda=11.8 torchvision torchaudio torchtext aws-ofi-nccl-dlc \
+  --strict-channel-priority --override-channels \
+  -c https://aws-ml-conda-pre-prod-sm.s3.us-west-2.amazonaws.com \
+  -c pytorch \
+  -c nvidia -c conda-forge
```

Comment:
Need to remove below file as aws conda package sets env vars for EFA instances, but DLC not just supports EFA instances and supports others as well

Example:
sudo rm -rf /opt/conda/pkgs/aws-ofi-nccl-$AWS_OFI_PLUGIN_VERSION-aws_0/etc/conda/activate.d/aws-ofi-nccl_activate.sh
sudo rm -rf /opt/conda/pkgs/aws-ofi-nccl-$AWS_OFI_PLUGIN_VERSION-aws_0/etc/conda/deactivate.d/aws-ofi-nccl_deactivate.sh

Reply:
Hi Jeet, do you know have the exact path to these bach scripts. I don't see these activate and deactivate scripts in the container.